docs/content/plugins/hashicorp.md

```diff
@@ -11,30 +11,60 @@ Source: https://github.com/dev-drprasad/porter-hashicorp-plugins
 
 ## Install or Upgrade
 
-**Note:** Supports porter version greater or equal to `v0.23.0-beta.1` and supports only `KV Version 2 secret engine`.
+Supports Porter version greater or equal to **v1.0.0-alpha.20**.
+Until the upstream plugin is updated to work with Porter v1, this is a [special release] with compatibility fixes.
+For older versions of Porter, use version v0.1.0 of the hashicorp plugin.
+We only support the [KV Version 2][kv-v2] secret engine. Please raise an issue if you're looking for support for other secret engines.
 
 ```
-porter plugin install hashicorp --feed-url https://github.com/dev-drprasad/porter-hashicorp-plugins/releases/download/feed/atom.xml
+porter plugin install hashicorp --version v0.1.0-porter.1 --url https://github.com/getporter/hashicorp-plugins/releases/download
 ```
 
+[special release]: https://github.com/getporter/hashicorp-plugins/releases/tag/v0.1.0-porter.1
+[kv-v2]: https://www.vaultproject.io/api-docs/secret/kv/kv-v2
+
 ## Plugin Configuration
 
 To use vault plugin, add the following config to porter's config file (default location: `~/.porter/config.toml`). Replace `vault_addr`, `vault_token` and `path_prefix` with proper values.
 
+The example below retrieves the vault_token from the VAULT_TOKEN environment variable.
+Do not store sensitive data in the Porter configuration file.
+
 ```toml
 default-secrets = "porter-secrets"
+
 [[secrets]]
-name = "porter-secrets"
-plugin = "hashicorp.vault"
+  name = "porter-secrets"
+  plugin = "hashicorp.vault"
 
-[secrets.config]
-vault_addr = "http://vault.example.com:7500"
-path_prefix = "organization/team/project"
-vault_token = "token"
+  [secrets.config]
+    vault_addr = "http://vault.example.com:7500"
+    path_prefix = "organization/team/project"
+    vault_token = "${env.VAULT_TOKEN}"
 ```
 
 ## Config Parameters
+#### path_prefix
+
+`path_prefix` lets allow you to specify prefix for your secret path.
```

Comment:
```suggestion
`path_prefix` allows you to specify a prefix for your secret path.
```

Reply:
Vaughn one day I will learn how to English but today is not that day! Thanks for being the best editor ever. ❤️